Gruntfile.js

```diff
@@ -21,35 +26,52 @@ module.exports = function(grunt) {
     pkg: grunt.file.readJSON('bower.json'),
 
     /**
-     * Set some src and dist location variables.
+     * Set directory location variables.
      */
     loc: {
-      src: './src',
+      src:  './src',
       dist: '.',
+      lib:  grunt.file.readJSON('./.bowerrc').directory,
       test: './_tests'
     },
 
     /**
      * Bower: https://github.com/yatskevich/grunt-bower-task
      *
-     * Install Bower packages and migrate static assets.
+     * Migrate static assets to distribution directory.
      */
     bower: {
-      install: {
+      process: {
         options: {
-          targetDir: './vendor/',
-          install: true,
-          verbose: true,
-          cleanBowerDir: true,
-          cleanTargetDir: true,
-          layout: function(type, component) {
-            if (type === 'img') {
-              return path.join('../static/img');
-            } else if (type === 'fonts') {
-              return path.join('../static/fonts');
-            } else {
-              return path.join(component);
+          targetDir: '<%= loc.lib %>',
+          install: false,
+          cleanTargetDir: false,
+          // The `exportsOverride` section in `bower.json` is used to map
+          // bower package components to types (js, css, less, img, fonts),
+          // which can then be moved in the `layout` function below.
+          //
+          // TODO: Switch to maintained grunt bower task.
+          // There is a bug in the `grunt-bower-task` which means
+          // certain packages have to appear in the `exportsOverride`
+          // section, even though they are only being copied in place.
+          // This has occurred with the `history.js`,
+          // and `string_score` packages. The consequence of this is that
+          // they are listed in `bower.json` and in the `browser` field
+          // in `package.json` when it shouldn't be necessary to list them
+          // in both places.
+          // Unfortunately, the `grunt-bower-task` is not being maintained
+          // as of May, 2015, so this likely will not be fixed soon.
+          // See background in issue:
+          // https://github.com/yatskevich/grunt-bower-task/issues/159
+          layout: function(type, component, source) {
```

Comment:
I was thinking I'd lint the whole Gruntfile in a separate PR after this is merged so the diff isn't crazy along with the functional changes, but I can get this particular change going because it's rewritten.